### PR TITLE
Create index.md

### PR DESCRIPTION
**Pull request by Smartmind12**
Adding a new index for Getting started page to start anchoring process for the broken links

Related to #48

### CODE REVIEW

File: docs/guides/index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+layout: page
+title: Guides
+---
+
+## Guides
```
Comment (postmodern) [inline docs/guides/index.md:6]: Should be `# Guides`.

**Review by postmodern — CHANGES_REQUESTED** [review]
Use `# Title` for top-level titles.

File: docs/guides/index.md (hunk 1 above)
Comment (Smartmind12) [reply docs/guides/index.md]: Done!

**Review by postmodern — APPROVED** [review]

**Comment by postmodern** [comment]
Hacktoberfest 2023 is now over! Congratulations on successfully submitting a PR to Ronin. As mentioned in the [blog post](https://ronin-rb.dev/blog/2023/09/26/ronin-is-participating-in-hacktoberfest-2023.html), you are now eligible for **free** [Ronin stickers](https://merch.ronin-rb.dev/category/stickers)! In order to claim your free stickers, please join the [Ronin Discord server](https://discord.gg/Cw5WWnycRH), and DM me the following information:

1. Which stickers do you want?
2. Your full mailing address.

The stickers will then be mailed in a regular flat-rate envelope, from the US, without tracking. Note, it may take up to one month for International mail to arrive.

**Pull request merged**